pipelines/nyu/adapters/course_catalog/src/process_data.rb

```diff
@@ -1,60 +1,101 @@
 require 'pry'
+require 'uri'
 
 # List of courses for a subject
 def extract_listings html
-	elements = html.children
-	elements[0] = Hash.new { |h,k| h[k] = [] }
-  binding.pry
-	# elements.inject do |first, second|
-	# 	nil
-	# end
-	nil
+  elements = html.children[1].child.children[1..]
```

Comment:
I think this method would be cleaner if selectors or xpaths were used

Reply:
Should be good now, replaced the above with the selector `css('div.primary-head ~ *')`